VargasMartinezJose/Dia3/HackerRank/Animal.java

```diff
@@ -0,0 +1,5 @@
+class Animal {
```

Comment:
Perfect! one file per class

But, what about packages ??? The default unnamed package should not be used

Try to re-arrange your classes into packages like

com.bootcamp.dayone......

or 

com.excercises.inheritance.one